packages/form-js-editor/test/spec/FormEditor.spec.js

```diff
@@ -417,75 +417,145 @@ describe('FormEditor', function() {
   });
 
 
-  describe('focus / blur events', function() {
+  describe('properties panel', function() {
 
-    const schema = {
-      type: 'default',
-      components: [
-        {
-          type: 'text',
-          label: 'Text',
-          key: 'text'
-        }
-      ]
-    };
+    describe('selection behavior', function() {
 
+      const schema = {
+        type: 'default',
+        id: 'Form_1',
+        components: [
+          {
+            type: 'text',
+            id: 'text',
+            label: 'Text',
+            key: 'text'
+          }
+        ]
+      };
 
-    it('should emit event on properties panel focus', async function() {
+      function expectSelected(expectedId) {
+        const panelEl = document.querySelector('.fjs-properties-panel');
 
-      // given
-      const formEditor = await createFormEditor({
-        schema,
-        container
-      });
+        const selectedId = panelEl.dataset.field;
+
+        expect(selectedId).to.eql(expectedId);
+      }
 
-      const focusinSpy = sinon.spy();
 
-      formEditor.on('propertiesPanel.focusin', focusinSpy);
+      it('should show schema per default', async function() {
+
+        // when
+        const form = await createFormEditor({
+          container,
+          schema
+        });
 
-      let input;
+        // assume
+        expect(form.get('selection').get()).not.to.exist;
 
-      await waitFor(async () => {
-        input = await screen.getByLabelText('Text');
+        // when
+        await wait();

```

Comment:
Is there a specific reason why we're introducing a custom `wait` here instead of using `waitFor` which only waits as long as necessary?

Reply:
Yes. The later simply does not work. :see_no_evil:

Reply:
It does: https://github.com/bpmn-io/form-js/pull/137/commits/2675e511bb76de68a243c23db10b1bb1ab53f9da 🙈